app/templates/_package.json

```diff
@@ -15,7 +15,13 @@
   },
   "scripts": {
     "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
-  },
+  },<% if (coveralls) { %>
+  "nyc": {
```

Comment:
I think non-official package.json properties should be at the bottom so it's clear they are that.